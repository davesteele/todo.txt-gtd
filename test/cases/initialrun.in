First task in the 'A' project @computer +AProj
First task in the 'B' project @phone +BProj
Second task in the 'A' project @computer +AProj
No project listed @computer
