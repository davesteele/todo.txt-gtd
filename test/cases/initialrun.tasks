
@computer
First task in the 'A' project @computer +AProj
Second task in the 'A' project @computer +AProj
No project listed @computer

@phone
First task in the 'B' project @phone +BProj
