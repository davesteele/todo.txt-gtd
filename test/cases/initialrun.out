# AProj
#

First task in the 'A' project @computer +AProj
Second task in the 'A' project @computer +AProj

# BProj
#

First task in the 'B' project @phone +BProj

# _None
#

No project listed @computer
